manifests/g/gif/ski/1.14.4/gif.ski.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: gif.ski

```

```suggestion
PackageIdentifier: ImageOptim.gifski
```
Please update file name and path to match this change.